test/test_core.py

```diff
@@ -9652,6 +9652,19 @@ def test_main_reads_args(self):
   def test_native_promise(self):
     self.do_core_test('test_native_promise.c')
 
+  def test_itimer(self):
+    self.do_core_test('test_itimer.c')
+
+  @node_pthreads
+  def test_itimer_pthread(self):
+    self.do_core_test('test_itimer.c')
+
+  @node_pthreads
+  def test_itimer_proxy_to_pthread(self):
+    self.set_setting('PROXY_TO_PTHREAD')
+    self.set_setting('EXIT_RUNTIME')
+    self.do_core_test('test_itimer.c')
```

Comment:
This feels a little heavy to me, three new tests in core that will run in all opt modes - do you expect optimizations to matter here? If not then perhaps they could be in `other`?

Reply:
Done